src/rp2_common/hardware_clocks/clocks.c

```diff
@@ -118,7 +163,11 @@ bool clock_configure(enum clock_index clk_index, uint32_t src, uint32_t auxsrc,
 }
 /// \end::clock_configure[]
 
-void clocks_init(void) {
+void clocks_init() {
```

Comment:
Should the `void` be left in the parentheses, as it was originally? (IIRC some compilers get grumpy without this)